src/datasources/alerts-api/tenderly-api.service.spec.ts

```diff
@@ -12,14 +12,10 @@ const networkService = {
 } as unknown as INetworkService;
 const mockNetworkService = jest.mocked(networkService);
 
-const httpErrorFactory = {
-  from: jest.fn(),
-} as unknown as HttpErrorFactory;
-const mockHttpErrorFactory = jest.mocked(httpErrorFactory);
-
 describe('TenderlyApi', () => {
   let service: TenderlyApi;
   let fakeConfigurationService: FakeConfigurationService;
+  const httpErrorFactory = new HttpErrorFactory();
```

Comment:
I would instantiate this on the `beforeEach` since if this class gets a state of some sort in the future, that state will be shared across each test which might make some of them fail.

Reply:
It's now reinstantiated in `beforeEach` as of 8125503817eab4802917485d9c30ac52169a9040.